src/components/reference/_cli-help-scan-output.md

```diff
@@ -336,10 +336,10 @@ OPTIONS
            option forces the project root to be a specific folder and assumes
            a local project without version control (novcs). This option is
            useful to ensure the '.semgrepignore' file that may exist at the
-           project root is consulted when the scanning root is not the
+           project root is consulted when the scan root is not the
```

Comment:
@mjambon FYI this is autogenerated from the :latest docker image's CLI --help output, so this will be overwritten unless the help text is also updated. Just an FYI.

Reply:
Good to know. I'll get to it when week or so.